src/components/stockDiscoveryComponents/stockSummary/stockSummary.js

```diff
@@ -31,19 +32,19 @@ function StockSummary() {
         <>
         {loading ? <LoadingSpinner /> : error  ? <MessageAlert variant='danger'>{error}</MessageAlert> :
         <>
-        <h3 className="stockdiscoveryRow">Today's Biggest Positive Movers</h3>
+        <h3 className="stockdiscoveryRow">Today's Biggest Positive Movers<ChevronRight /></h3>
```

Comment:
These confused me a bit when I seen them on frontend I taught the heading was now clickable and the arrow was saying to go somewhere. I think they will get that bar at bottom means scroll. 
![image](https://user-images.githubusercontent.com/61060096/200590177-677a924d-08ef-4763-8bb8-96cfcf4ec19c.png)

It also looks bit odd on mobile I think. 
![image](https://user-images.githubusercontent.com/61060096/200590331-da175acb-02f2-4d83-8f48-46035c65626e.png)